Entities/Product.java

```diff
@@ -0,0 +1,67 @@
+package Entities;
+
+import java.util.List;
+
+public class Product {
+    //Attributes
+    private Integer productId;
+    private String name;
+    private Double price;
+    private String category;
+    private List <String> ingredients;
```

Comment:
Ingredients should not be a list of strings, since the ingredients itself might contain properties you'd need.

Reply:
done